index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/5706a27a8e.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section id="main">
+        <section class="statusBar bcGrey">
+            <div class="statusBarContent">
+                <div class="time"><img src="img/Time.png" alt=""></div>
+                <div class="statusIcons">
+                    <img src="img/Cellular.png" alt="">
+                    <img src="img/Wifi.png" alt="">
+                    <img src="img/Battery.png" alt="">
+                </div>
+            </div>
+        </section>
+        <section class="navigation bcGrey">
+            <div class="navigationContent">
+                <p class="roboto white w700 s18 lh24">John Doe</p>
+                <nav>
+                    <img src="img/Union.png" alt="">
+                </nav>
+            </div>
+        </section>
+        <section class="headline">
+            <div class="headlineContent">
+                <h1 class="crete w400 s56 lh64 ls037 orange width327">Hey There. <br>I'm Alex </h1>
+                <p class="roboto w700  s24 lh32 white1 width327">I am a Software Developer</p>
+                <p class="poppins w400 s16 lh24 white width327">I can help you build a product, feature or website, Look through some of my work and experience! If you like what you see and have a project
+                    you need coded, don't hesitate to contact me.</p>
+                <div>
+                    <a href="#"><img src="img/gitHub.png" alt=""></a>
+                    <a href="#"><img src="img/linkedin.png" alt=""></a>
+                    <a href="#"><img src="img/angellist.png" alt=""></a>
+                    <a href="#"><img src="img/twitter.png" alt=""></a>
+                    <a href="#"><img src="img/medium.png" alt=""></a>
+                </div>
+            </div>
```

Comment:
- [ ] Kindly group all your `social icons` in a `ul` tag and each of them in a `li` tag 🙏 . This will make your code semantically correct ☺️ 
Example:
```
<ul>
  <li>
    <a>
      <img>
    </a>
  </li>
</ul>
```

Reply:
Fixed.